singleuser/user-config.py

```diff
@@ -3,4 +3,11 @@
 mylang = 'test'
 family = 'wikipedia'
 
-usernames['wikipedia']['test'] = os.environ['JPY_USER']
+
+custom_path = os.path.expanduser('~/user-config.py')
```

Comment:
I think this might need to be unset somehow? Leaving random variables in user-config.py issues a warning:
```
pywikibot/config2.py:1009: UserWarning: Configuration variable foobar is defined but unknown. Misspelled?
```